### ISSUE TITLE
Regression: Docker container does not start with v1.1.0

### ISSUE THREAD

**Issue by ledermann** [plausible/analytics#360]
## Prerequisites
- [x] I have searched open and closed issues to make sure that the bug has not yet been reported.

## Bug report

After updating the Docker image after v1.1.0 was released, the container cannot be started anymore. The logs are:

```
Loading plausible..
Starting dependencies..
Starting repos..
create Plausible.Repo database if it doesn't exist
create Plausible.ClickhouseRepo database if it doesn't exist
eation of Db successful!
/entrypoint.sh: line 11: exec: db: not found
```

Seems to be caused by #352. Ping @BobyMCbobs.

Sadly, a rollback is not simple because on Docker Hub there are no tags:
https://hub.docker.com/r/plausible/analytics/tags

To fix this issue for me, I checked-out the sources from [v1.1.0](https://github.com/plausible/analytics/releases/tag/v1.1.0), reverted 59acd4c76d6ce77e00d3af46d4aea20c1d33586e, rebuild the image ~and pushed it to [ledermann/plausible-fixed](https://hub.docker.com/r/ledermann/plausible-fixed)~. This image works fine for me - I will delete it after there is a fixed official image. 
**Update:** Just deleted my image, because the official image is fixed.

**Comment by ukutaht** [comment]
Reverting now, will release a patch update ASAP

**Comment by ukutaht** [comment]
Can you try the `latest` tag now

**Comment by ledermann** [comment]
The image `plausible/analytics:latest` works fine now 👍

**Comment by bilyboy785** [comment]
Same issue here with latest tag

**Comment by BobyMCbobs** [comment]
My apologies for the mishap. In local development I don't recall this issue.

This change leaves my issue (#352) unaddressed.
I will investigate further.

**Comment by BobyMCbobs** [comment]
@ukutaht and @ledermann,

please try https://github.com/plausible/analytics/pull/362

**Comment by ukutaht** [comment]
@bilyboy785 Can you run the `latest` or `v1.1.1` tag?

**Comment by ukutaht** [comment]
This should be fixed in `v1.1.1`. Closing
